custom_components/google_home/sensor.py

```diff
@@ -105,6 +117,13 @@ def get_device_attributes(device):
         }
 
 
+class AlarmsAttributes(TypedDict):
+    """Typed dict for alarms attributes"""
+
+    alarms: List[GoogleHomeAlarmDict]
+    integration: str
```

Comment:
I suggest we move out `TypedDict` to a separate file like `types.py`. It would give it a bit better structure